adhocteam/Head-Start-TTADP
# Add loader

## Description of change

Add a loading overlay to be shown when components are loading. Added the loader to the `Container` component so it dims the complete container when loading. Widgets now have default data defined so they can be displayed with "null" data. This should stop the UI from "jumping" around so much as data is loaded in and components are resized.

Removed some old code that is no longer needed (data scripts that have already been ran against prod).

## How to test

Browse the regional dashboard on [sandbox](https://tta-smarthub-sandbox.app.cloud.gov/regional-dashboard). Note when you change filters or load the page a faint white overlay is shown and elements inside are not click-able. The tables/widget on the activity reports page also has the white overlay when new data is being loaded.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-152

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
